dali/pipeline/operators/crop/crop.cc

```diff
@@ -40,7 +40,8 @@ void Crop<CPUBackend>::DataDependentSetup(SampleWorkspace &ws) {
 
   const DALITensorLayout in_layout = input.GetLayout();
   DALI_ENFORCE(in_layout == DALI_NHWC || in_layout == DALI_NCHW
-            || in_layout == DALI_NFHWC || in_layout == DALI_NFCHW,
+            || in_layout == DALI_NFHWC || in_layout == DALI_NFCHW
+            || in_layout == DALI_NDHWC,
```

Comment:
Open question: Should I introduce DALI_NCDHW as well?

Reply:
I'd say yes - it's quite likely that we'll get this format from MRI scans.